OrbitGl/TimeGraph.cpp

```diff
@@ -100,7 +99,7 @@ TimeGraph::TimeGraph() { m_LastThreadReorder.Start(); }
 
 //-----------------------------------------------------------------------------
 void TimeGraph::SetStringManager(std::shared_ptr<StringManager> str_manager) {
-  string_manager_ = str_manager;
+  string_manager_ = std::move(str_manager);
```

Comment:
hmm... this is a shared_ptr, what does std::move archive here?

Reply:
"Clang-Tidy: Parameter 'str_manager' is passed by value and only copied once; consider moving it to avoid unnecessary copies". In this case (in theory) it saves the reference counter from being incremented and decremented right away.